kolibri/utils/conf.py

```diff
@@ -45,6 +45,7 @@
     "kolibri.plugins.management",
     "kolibri.plugins.learn",
     "kolibri.plugins.document_pdf_render",
+    "kolibri.plugins.html5_app_renderer",
```

Comment:
I saw some comments (I think by @indirectlylit) elsewhere about switching things towards "renderer" (which is more grammatical, though more awkward), but not sure whether that was preferred throughout now. Can switch as desired.